content/templates/output-formats.md

```diff
@@ -142,12 +152,19 @@ outputs:
   page: ["HTML"]
 ```
 
+In the above examples, the *output formats* for `section`, `taxonomyTerm` and
+`taxonomy` will be implicitly set to `["HTML"]`.
+
+{{% note %}}
+If you add an `outputs` config section, the *output format* for all the
+unspecified *Kinds* will be set to only `HTML`.
```

Comment:
I think you take the "half-empty" approach to this. I would much prefer something in the line of:

>If you add an `outputs` config section, you replace the Hugo default `output` configuration with your own and must provide a complete configuration. You will get `HTML` for any missing page `Kind` entry.

Reply:
OK

Reply:
Done (this, and also the next hunk where you commented).